docs/localnet.md

```diff
@@ -1,16 +1,34 @@
 # Local Testnets
 
-Prerequisites:
+Setting up a Fendermint local testnet is a way to get started quickly with IPC.
+
+This guide offers two flavours:
+
+- A single node deployment: useful for developing smart contracts and testing the APIs.
+- A 4 node testnet: useful for testing consensus, checkpointing, and more.
+
+## Prerequisites
+
+On Linux (links and instructions for Ubuntu):
+
+- Install system packages: `sudo apt install build-essential clang cmake pkg-config libssl-dev protobuf-compiler`.
```

Comment:
We should maybe add this to a `make install-deps` to make users lives easier. I will track it in an issue.

Reply:
Can do, but this step is OS and distribution-dependent.

- Would need to check compatibility and abort if an unknown/unsupported Linux distro is being used.
- Would be great to make it support macOS.

Having said that, I'm migrating these scripts to use docker images, so this step will soon become obsolete/unnecessary.